app/actions/websocket/channel.ts

```diff
@@ -46,16 +104,27 @@ export async function handleUserRemovedEvent(serverUrl: string, msg: any) {
                 if (await isTablet()) {
                     const channelToJumpTo = await queryLastChannelFromTeam(database.database, channel?.teamId);
                     if (channelToJumpTo) {
-                        switchToChannel(serverUrl, channelToJumpTo);
+                        const {models: switchChannelModels} = await switchToChannel(serverUrl, channelToJumpTo);
```

Comment:
is this correct? if it is, you are missing the last argument set to true so that `switchToChannels` only prepares and not writes to the db

Reply:
Yes. My plan is to do all the database operations at once, included those related to switchToChannel. I fixed the missing argument, but feel free to let me know if you see any other potential problem with this approach.